src/styles/variables.less

```diff
@@ -11,4 +9,39 @@
 
 @knappBredde: 13em;
 
-@navBorder: @navGra20;
+@navBorder: var(--navds-semantic-color-border); // works?
+@navBakgrunn: #ffffff;
+
+// https://github.com/navikt/nav-frontend-core/blob/162c248da84041fa91bbcd8eadb8449bc90e8556/less/_variabler.less
```

Comment:
Trengs alle disse fremdeles? Hvordan er migreringsplanen?

Reply:
Nei, ikke alla trengs. Skal gå igenom de og fjerne de som ikke bruks/trengs innan jag merger. 

Finns tokens for de nye fergene i ds-react biblioteket, så egentligen er planen å ta bruk de fergene som står närmst det vi har idag (de samsvarar ikke helt alltid).